src/modules/Host.rb

```diff
@@ -156,6 +165,7 @@ def Write(gui: false)
     # @return true if success
     def Import(settings)
```

Comment:
Have not checked all the code and the bug comments yet, but wasn't enough just returning false before marking it as initialized in case of no "hosts" section?
```suggestion
     def Import(settings)
       return false unless settings.key("hosts")
```

Reply:
That has been the first try but it has failed. The import will not be called at all if the section "host" is not in AY configuration file. And with this call the user would not be able to delete all host entries. Not regarding if it is useful or not :-)

Reply:
If that it is the case then the next check is like not necesary
```ruby
       @configuration_imported = true # if settings.key?("hosts")
```